package.json

```diff
@@ -65,31 +65,15 @@
 			"tsx"
 		],
 		"rules": {
+			"import/no-unresolved": "off",
 			"import/no-unassigned-import": "off",
-			"no-unused-vars": [
-				2,
-				{
-					"varsIgnorePattern": "^React$"
-				}
-			]
+			"@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
@bfred-it Are you ok with making all void function explicit?

Meaning `async function init() {` => `async function init(): void {`

This will prevent us from accidentally return something.

Reply:
Ok, I'll leave that for a separate PR. It's quite a big change.

Reply:
@nickytonline do you have time to drop this rule?

Reply:
I'll look at it later tonight.